src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const smallImages = document.querySelectorAll('.gallery__thumb');
+const bigImage = document.querySelector('.gallery__large-img');
+
+smallImages.forEach(image => {
+  image.addEventListener('click', (element) => {
+    element.preventDefault();
+
+    if (element.target.tagName === 'IMG') {
```

Comment:
Instead of checking the tagName, you can use the [closest](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest)() method to find the nearest ancestor element that matches a given selector. This way, you won't be limited to specific tag names and can use a more flexible approach.